applications/MeshMovingApplication/tests/test_ale_fluid_solver/ProjectParameters_ale_on_subdom_struct_sim_mono.json

```diff
@@ -12,7 +12,11 @@
         "ale_boundary_parts" : ["circ_inner", "rect_inner"],
         "mesh_motion_parts"           : ["circ_mesh", "rect_mesh"],
         "mesh_motion_solver_settings" : {
-            "solver_type" : "structural_similarity"
+            "solver_type" : "structural_similarity",
+            "calculate_mesh_velocity": true,
+            "mesh_velocity_calculation": {
+                "time_scheme": "bossak"
```

Comment:
this must be selected automatically, this was one of the big features in the ale-fluid solver

Don't worry, I will take a look myself and fix it

Reply:
Actually I had problems with this test case. The point is that by default the mesh calculation is done with the BDF2 formulas (as we used to do a long time ago).

Reply:
I know, I designed it to give problems if the behavior changes ;)

As I said, I will fix it, no worries

Reply:
In my FSI solver I enforced the ALE and the CFD schemes to be the same for the sake of consistency. If you want I can do the same in the ale_fluid_solver.py too.

Reply:
Thats actually how it is now

Sure if you know how to change it please go ahead

not that this is done in the [derived class in the fluid-app](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/FluidDynamicsApplication/python_scripts/navier_stokes_ale_fluid_solver.py)

Can you please also check the [one in the PotentialFlowApp](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CompressiblePotentialFlowApplication/python_scripts/ale_potential_flow_solver.py)?

Reply:
I think that last commit is a good solution to handle this.